frontend/src/App.css

```diff
@@ -14,14 +14,28 @@
 }
 
 .App-header {
-  background-color: #282c34;
+ /* background-image: url("https://en.cryptonomist.ch/wp-content/uploads/2022/08/nft-hacker-1140x600.jpg.webp");
+  background-size: cover;
+  height: 110vh;
+  margin-top: 50px;
+  min-width: 90px;
+  background-repeat:no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  /*justify-content: center;
+  /*background-color: #282c34;
   min-height: 100vh;
   display: flex;
   flex-direction: column;
```

Comment:
You have to fix this css file. Delete the things you do not need. No commented out code goes into the master/main branch.